src/notifications/tests/test_models.py

```diff
@@ -63,3 +65,120 @@ def new_notification(sender, **kwargs):
+    """ 
+    # HELPER FUNCTIONS
+
+    def create_notification(self, text="Sample Text") -> Notification:
+        """ 
+            Quick helper function that spits out different notifications depending on text var
+        """
+        comment = baker.make("comments.Comment", text=text)
+        notification = Notification.objects.create(
+            recipient=self.student,
+            sender_object=self.admin,
+            verb="",
+
+            action_object=comment,
+            action_object_id=comment.pk,
+            action_content_type=self.comment_content_type,
+
+            target_object=self.quest_submission,
+            target_object_id=self.quest_submission.pk,
+            target_content_type=self.quest_submission_content_type,
+
+            font_icon="<i class='fa fa-info-circle'></i>",
+        )
+        return notification
+
+    def notification_start_text(self, notification_id) -> str:
+        """  
+            Recreates string tag for testing purposes. Equivalent to: str(notification) up until action_object
+            '{notification_start_text(id)}{action_object}{notification_end_text}' <=> str(notification)
+        """
+        site_url = 'https://tenant.test.com'
+        notification_url = reverse('notifications:read', args=[notification_id])
+        submission_url = reverse('quest_manager:submission', args=[self.quest_submission.pk])
+        chars = f"{self.admin.username}  <a href='{site_url}{notification_url}?next={submission_url}'> <em>{self.quest.name}</em> with \""
+
+        return chars
+
+    def notification_text(self, test_case) -> str:
+        """ 
+            Helper for tests. Just here to prevent extra lines of boilerplate code.
+            This just gets the comment text from notifications model and returns it
+        """ 
+        notification = self.create_notification(text=test_case)
+        strip_count = len(self.notification_start_text(notification.pk))
+
+        # this is just to get the text without the enclosing <a> tag
+        notification_text = str(notification)[strip_count:-len('"</a>')]
+
+        return notification_text
+
+    # 
+
+    def setUp(self):
+        User = get_user_model()
+        
+        self.admin = User.objects.get(username="admin")
+        self.student = baker.make(User, username="student")
+        self.quest = baker.make("quest_manager.Quest", name="Quest")
+        self.quest_submission = baker.make("quest_manager.QuestSubmission", user=self.student, quest=self.quest)
+
+        self.comment_content_type = ContentType.objects.get(app_label="comments", model="comment")
+        self.quest_submission_content_type = ContentType.objects.get(app_label="quest_manager", model="questsubmission")
+
+    #  Check if notification string method leaves behind un-stripped html tags except <img>.
+    
+    def test_notification_str__check_with_no_html(self):
+        """ 
+            Check with no html (no stripped tags) 
```

Comment:
Check what? What are stripped tags? Maybe provide examples in the docstring.
My guess:  "Check that a notification comment(?) without htmls tags is not changed by __str__()"  Is that what you're checking here? I'm not sure because you don't actually call `str(notification)` anywhere?

Reply:
str(notif) is called in one of the helper functions
![image](https://user-images.githubusercontent.com/39788517/175724181-fd713107-a2bf-4725-b109-e11e1400a1b9.png)

Reply:
clarified doc string